tests/e2e/default/login/renew-token.wdio-spec.js

```diff
@@ -8,29 +8,29 @@ describe('should renew token', async () => {
 
   beforeEach(async () => {
     await loginPage.cookieLogin();
+    await commonPage.waitForPageLoaded();
   });
 
   it('Refresh page multiple times and verify token gets renewed', async () => {
-    await commonPage.waitForPageLoaded();
-
     for (let counter = 0; counter < 3; counter++) {
-      const beforePageLoadTime = moment().add(utils.ONE_YEAR_IN_S, 'seconds');
+      const beforePageLoadTime = moment().add(utils.ONE_YEAR_IN_S, 'seconds').startOf('second');
       await browser.refresh();
       await commonPage.waitForPageLoaded();
-      const afterPageLoadTime = moment().add(utils.ONE_YEAR_IN_S, 'seconds');
+      const afterPageLoadTime = moment().add(utils.ONE_YEAR_IN_S, 'seconds').endOf('second');
       const ctxExpiry = await getCtxCookieExpiry();
 
       expect(
-        ctxExpiry.isBetween(beforePageLoadTime, afterPageLoadTime),
+        ctxExpiry.isBetween(beforePageLoadTime, afterPageLoadTime, 'second', '[]'),
         `Failed for counter = ${counter}, ${ctxExpiry} ${beforePageLoadTime} ${afterPageLoadTime}`
       ).to.be.true;
+      await browser.pause(1000);
```

Comment:
The cookie expiry precision is second, not microsecond. We need to wait for a full second to make the test valuable.